zipline/src/hostMain/kotlin/app/cash/zipline/EventListener.kt

```diff
@@ -72,6 +72,21 @@ abstract class EventListener {
     return null
   }
 
+  /**
+   * Invoked when an application load was skipped because the code is unchanged.
+   *
+   * @param info additional info on the load-skipping
+   * @param startValue the value returned by [applicationLoadStart] for the start of this call. This
+   *   is null unless [applicationLoadStart] is overridden to return something else.
+   */
+  open fun applicationLoadSkipped(
+    applicationName: String,
+    manifestUrl: String?,
+    info: String,
```

Comment:
Is there a more typesafe way to do this?

Reply:
definitely, what are you thinking about? (there are different ways!)

Reply:
ok, realized this: https://github.com/cashapp/zipline/pull/1199#discussion_r1430487746